src/test/resources/features/99d_employment_qa_smoke.feature

```diff
@@ -73,129 +73,128 @@ Feature: Smoke tests for end of year Employment Journeys to DES/IF in QA - Incom
+    And the user clicks the continue button
+    Then the user should be redirected to the "Did your employer pay any of your Income Tax or incurred costs?" page
+    When the user selects the no radio button
+    And the user clicks the continue button
+    Then the user should be redirected to the "Did you get any reimbursed costs, vouchers or non-cash benefits from your employer?" page
+    When the user selects the no radio button
+    And the user clicks the continue button
+    Then the user should be redirected to the "Did you get any assets from this company?" page
+    When the user selects the no radio button
+    And the user clicks the continue button
+    Then the user should be redirected to the "Check your employment benefits" page
+    And the user clicks the continue button
+    Then the user should be redirected to the "Do you want to claim employment expenses?" page
+    When the user selects the yes radio button
+    And the user clicks the continue button
+    Then the user should be redirected to the "Employment expenses" page
+    When the user clicks the continue button
+    Then the user should be redirected to the "Do you want to claim business travel and overnight expenses?" page
+    When the user selects the yes radio button
+    And the user clicks the continue button
+    Then the user should be redirected to the "How much do you want to claim for business travel and overnight stays?" page
+    When the user selects the amount field and enters a value of 100
+    And the user clicks the continue button
+    Then the user should be redirected to the "Do you want to claim for uniforms, work clothes, or tools?" page
+    When the user selects the no radio button
+    And the user clicks the continue button
+    Then the user should be redirected to the "Do you want to claim for professional fees and subscriptions?" page
+    When the user selects the no radio button
+    And the user clicks the continue button
+    Then the user should be redirected to the "Do you want to claim for buying other equipment?" page
+    When the user selects the no radio button
+    And the user clicks the continue button
+    Then the user should be redirected to the "Check your employment expenses" page
+    And the user clicks the save and continue button
+    Then the user should be redirected to the "PAYE employment" page
+    When the user clicks on the remove employment link
+    Then the user should be redirected to the "Are you sure you want to remove this employment?" page
+    When the user clicks the Remove employer button
+    Then the user should be redirected to the "Do you want to add an employer?" page
+    When the user selects the no radio button
+    And the user clicks the continue button
+    Then the user should be redirected to the "Your Income Tax Return" page
 
   Scenario: Agent user with at least one prior employment adds a new Employment
     When the user logs into the service with the following details
```

Comment:
this test is currently failing at the student loans journey:
`"Did you[r client] repay any student l..." was not equal to "Did you[] repay any student l..."`

Reply:
this was resolved in another PR